test_common/fixtures/personas.yml

```diff
@@ -28,11 +28,11 @@ company01:
   disagree_claimants_job_or_title: 'lorem ipsum job title'
   # Earnings and Benefits
   agree_with_claimants_hours: 'No'
-  queried_hours: 32
+  queried_hours: 32.0
```

Comment:
This looks like you have tweaked your personas to suit something else.  Unless a user really would key in 32.0 then this should just be 32 as before ?  If your code needs a float then this should be converted elsewhere